server/resp/src/main/java/org/infinispan/server/resp/RespServer.java

```diff
@@ -40,7 +41,20 @@ protected void startInternal() {
       if (!cacheManager.getCacheManagerConfiguration().features().isAvailable(RESP_SERVER_FEATURE)) {
          throw CONFIG.featureDisabled(RESP_SERVER_FEATURE);
       }
-      if (cacheManager.getCacheConfiguration(configuration.defaultCacheName()) == null) {
+      super.startInternal();
+   }
+
+   @Override
+   protected void startCaches() {
+      String defaultRespCacheName = configuration.defaultCacheName();
+      Configuration respCacheConfig = cacheManager.getCacheConfiguration(defaultRespCacheName);
+      if (respCacheConfig != null) {
+         ExpirationConfiguration expConfig = respCacheConfig.expiration();
```

Comment:
I think we should kill this code: being able to set server-side expiration is an advantage we have over the real Redis